frontend/src/scenes/actions/ActionStep.js

```diff
@@ -328,11 +328,18 @@ export class ActionStep extends Component {
                             <this.AutocaptureFields step={step} isEditor={isEditor} actionId={actionId} />
                         )}
                         {step.event === '$pageview' && (
-                            <this.Option
-                                item="url"
-                                extra_options={<this.URLMatching step={step} isEditor={isEditor} />}
-                                label="URL"
-                            />
+                            <div>
+                                <this.Option
+                                    item="url"
+                                    extra_options={<this.URLMatching step={step} isEditor={isEditor} />}
+                                    label="URL"
+                                />
+                                {step.url_matching == 'contains' && (
```

Comment:
RE: Aaron's observation. OR'ing for default null of step.url_matching along with this condition will resolve!